addons/editor-buttons-reverse-order/userstyle.css

```diff
@@ -50,10 +50,13 @@
   border-bottom-left-radius: 0;
 }
 
-[class*="controls_controls-container"] img {
+[class*="green-flag_green-flag_"],
+[class*="stop-all_stop-all_"],
+.pause-btn {
   order: 1;
 }
 
-[class*="controls_controls-container"] div {
-  order: 0;
+.sa-debugger-container {
+  margin-left: 0.2rem;
+  margin-right: 0px !important;
 }
```

Comment:
Use separate rules for `[dir="ltr"]` and `[dir="rtl"]`. You can also move the addon to the "themes" section at the bottom of `addons.json` to make the `!important` unnecessary.

Reply:
I reverted the change to djdolphin's code because it worked fine before. I also made this compatible with the gamepad addon.